__tests__/app.test.js

```diff
@@ -13,153 +14,300 @@ beforeEach(() => {
+      expect(msg).toBe("Path not found");
+    });
+});
+
+test("Patch, /api/reviews/:review_id : status 200 ok, returns an object with updated votes field as an object.", () => {
+  const updateVote = {
+    inc_votes: 10,
+  };
+  return request(app)
+    .patch("/api/reviews/2")
+    .send(updateVote)
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.review).toEqual({
+        title: "Jenga",
+        designer: "Leslie Scott",
+        owner: "philippaclaire9",
+        review_img_url:
+          "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+        review_body: "Fiddly fun for all the family",
+        category: "dexterity",
+        created_at: "2021-01-18T10:01:41.251Z",
+        votes: 15,
+        review_id: 2,
       });
-      test("Testing for a 404 error with a invalid review_id that does not exist", () => {
-        return request(app)
-          .get("/api/reviews/9999")
-          .expect(404)
-          .then(({ body }) => {
-            expect(body.msg).toBe("Path not found");
-          });
+    });
+});
+test("Patch, /api/reviews/:review_id : status 200 ok, returns an object with updated votes field as an object. When entering negative numbers to inc_votes", () => {
+  const updateVote = {
+    inc_votes: -3,
+  };
+  return request(app)
+    .patch("/api/reviews/2")
+    .send(updateVote)
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.review).toEqual({
+        title: "Jenga",
+        designer: "Leslie Scott",
+        owner: "philippaclaire9",
+        review_img_url:
+          "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+        review_body: "Fiddly fun for all the family",
+        category: "dexterity",
+        created_at: "2021-01-18T10:01:41.251Z",
+        votes: 2,
+        review_id: 2,
       });
     });
-  });
+});
+test("Status 404 path not found, if review_id entered is valid but does not exist", () => {
+  const updateVote = {
+    inc_votes: 10,
+  };
+  return request(app)
+    .patch("/api/reviews/9999")
+    .send(updateVote)
+    .expect(404)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Path not found");
+    });
+});
+test("Status 400 path not found, if review_id entered the wrong data type", () => {
+  const updatedVote = {
+    inc_votes: 15,
+  };
+  return request(app)
+    .patch("/api/reviews/nan")
+    .send(updatedVote)
+    .expect(400)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Bad request");
+    });
+});
+test("Status 400 Bad request, needs valid new comment object with inc_votes key", () => {
+  const updateVote = {};
+  return request(app)
+    .post("/api/reviews/1/comments")
+    .send(updateVote)
+    .expect(400)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Bad request");
+    });
+});
+test("Status 400 Bad request, needs valid new comment with correct data type", () => {
+  const updateVote = { inc_votes: "sausages" };
```

Comment:
Love this, not only because it's a good test that people often forget, but also because "sausages"